### DISCUSSION TITLE
Issue with Particle Effect in React Project using react-tsparticles

### DISCUSSION THREAD

**Discussion by klvndrgn** [matteobruni/tsparticles#5257] [Q&A] [answered]
Hi everyone,

I'm currently facing an issue while attempting to utilize the particle effect from tsparticles in my React project. I've installed the necessary dependencies for particles, and here's the current version I have installed:

```
"particle.js": "^1.0.7",
"react-tsparticles": "^2.12.2",
"tsparticles": "^2.12.0"
```

I've imported all the required files to call the loadFull function, and during debugging, I logged the particles instance. Here's what I found:
![image](https://github.com/tsparticles/tsparticles/assets/76170148/f3c929ee-b187-4263-ad8c-bfde9acba37a)

My concern is whether react-tsparticles is no longer supported. Can anyone provide advice on how to proceed or suggest potential solutions?

Thanks in advance!

**Comment by matteobruni** [comment, marked as answer]
Please open an issue on the react repository https://github.com/tsparticles/react
